src/php/views/page.twig

```diff
@@ -1,9 +1,10 @@
 {% extends "layouts/base.twig" %}
 
 {% block content %}
-
 	<div class="obj-width-limiter">
 		{{ post.content }}
+		{# Loads comments.php by default #}
+		{{ function('comments_template') }}
```

Comment:
This seemed like the easiest way to make the default comment functionality in WordPress *just work*, and in fact, comments should display when this is added even without `comments.php` (or `comments.twig`), but it will give a warning that it doesn't like this. I'm a little wary of using this because it feels a little different than how we're including the rest of our templates, so I'm very happy to have any kind of feedback on this.

Reply:
I think this is fine, we include other WordPress-y templates in a similar way, like the search and password forms, so this isn't a huge departure.